portfolio/src/main/webapp/script.js

```diff
@@ -29,4 +29,18 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+}  
+
+  // Hello Sydney function
```

Comment:
Super nit on styling: un-indent it to make the line vertically align with the function below this line.